### DISCUSSION TITLE
Custom API Hook when sort items

### DISCUSSION THREAD

**Discussion by dsivokhin** [directus/directus#5180] [Feature Requests]
I wrote hook for sync with external service, it's work for CRUD events, but, when I sort items using drag&drop, I can see in console POST/utils/sort/:collection, how catch sort events for hook ?

![изображение](https://user-images.githubusercontent.com/4669992/115574137-096a7f80-a2ca-11eb-87f8-1a0d5c5cabbd.png)

**Comment by rijkvanzanten** [comment]
Good one! The sort util currently doesn't fire any events. Would you like to open a PR to add one?

**Reply by dsivokhin** [reply]
Thank you, PR I can't,  better it could be feature request.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
